backend/data/dataSources.py

```diff
@@ -69,12 +29,12 @@
         "type": "csv",
         "res": "LA",
         "enabled": True,
-        "path": "data/transformed/censusData_bias.csv",
+        "path": "backend/data/cleaned/censusData_bias.csv",
```

Comment:
There is still one **big** concern I have about this data structure: there is still a lack of a proper separation of concern: we're still mixing data and visualisation properties in the same structure. 

We need to separate them by proper encapsulation. 
As suggested somewhere else (in chat, maybe) we should go towards having this structure to combine `Dataset` and `LayerSpec` classes - that can also (potentially - and most likely) be re-used.

Reply:
In addition - and this is a bit a really don't like - it is **not** clear at glance what components are actually composing the Data Sources (list).

So, in terms of `typing` specifications:

```
DataSource <- Union[Dataset, LayerSpec]
DataSources <- List[DataSource]
```

In this way, it is **also** immediate to enable and disable steps we don't want. 
Atm it is not very clear to me how you disabled `geoupCount`?

Reply:
EDIT: 
having the group-meta layer can also allow for aggregation of the pipeline (or not).

By this I mean that a `LayerSpec` (or part of it) can either be associated to Group (i.e. presentation bit shared for all the data sources in that category).

Reply:
A **general** comment about data in cleaned: 

The _pipeline_ **must** be _fully reproducible** --> this means that it should be **always** be possible to go back and fro the transformations. 

In other words, every single file generated and saved in _cleaned_ must have (in the documentation and code) a clear _inverse transformation_ procedure. 

By that *I don't mean* we need to transform the _cleaned_ data back to the original one.
The _inverse transformation_ can be achieved at code-level bu the pair (`source raw file`, `transformation_pipeline`).

This is paramount important and we need to keep this always in mind.

Reply:
For this PR, pls let's make sure that all the new files in cleaned have the proper documentation about raw data source and reference to code that has generated the cleaned data file(s).

Reply:
Let's talk about this some time, I don't fully follow and feel we could explain better over zoom